src/Adapter/Product/CommandHandler/UpdateProductShippingHandler.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * 2007-2020 PrestaShop SA and Contributors
```

Comment:
Please use new license headers

Reply:
I've updated headers in my editor. 
Not gonna update them manually now, as Im aware there is a command for that, which will be run at some point later to update all headers isn't it?

Reply:
Indeed. @Progi1984 I regularly run the command to update the codebase and make sure it stays fine 😉